pkg/controller/podgc/gc_controller.go

```diff
@@ -118,6 +145,32 @@ func (gcc *PodGCController) gc() {
 	wait.Wait()
 }
 
+// cleanupOrphanedPods deletes pods that are bound to nodes that don't exist.
+func (gcc *PodGCController) gcOrphaned() {
+	glog.Infof("GC'ing orphaned")
+	pods, err := gcc.podStore.List(labels.Everything())
```

Comment:
Instead of listing the same pods twice, please change it to:
- list pods in gc() function
- pass this []*api.Pod to both functions